radixdlt-java/src/main/java/com/radixdlt/client/core/network/IncreasingRetryTimer.java

```diff
@@ -10,9 +10,22 @@
 public class IncreasingRetryTimer implements Function<Observable<Throwable>, ObservableSource<Long>> {
 	private static final Logger LOGGER = LoggerFactory.getLogger(IncreasingRetryTimer.class);
 
+	private final Function<Throwable, Boolean> filter;
+
+	public IncreasingRetryTimer(Function<Throwable, Boolean> filter) {
```

Comment:
Maybe consider using `Predicate<Throwable>` instead? As I understand, that's what it's for.